lib/components/root/home.vue

```diff
@@ -0,0 +1,103 @@
+<!--
+Copyright 2017 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/opendatakit/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div>
+    <div id="root-home-heading">
+      <h1>Welcome to Central.</h1>
+      <p>Let's get some things done.</p>
+    </div>
+    <loading :state="fetchCounter < 3"/>
+    <div v-show="fetchCounter === 3">
+      <div class="panel panel-simple">
+        <div class="panel-heading"><h1 class="panel-title">You Have:</h1></div>
+        <div class="panel-body">
+          <div class="row">
+            <div class="col-xs-6">
+              <root-entity :icons="['user-circle', 'cog']" name="Web User"
+                data-from="/users" route-to="/users"
+                @fetched="incrementCounter">
+                who are allowed to administer projects through this website.
```

Comment:
let's modify this to "who can administer" so we don't have to worry about pluralization.